data/Quake Champions

```diff
@@ -0,0 +1,17 @@
+Anarki
+Athena (QC)
```

Comment:
Does QC make sense or should it be Quake? I assume there is only one Athena and one Ranger in Quake?